addons/delivery/models/delivery_carrier.py

```diff
@@ -87,7 +87,7 @@ def install_more_provider(self):
             'name': 'New Providers',
             'view_mode': 'kanban,form',
             'res_model': 'ir.module.module',
-            'domain': [['name', 'ilike', 'delivery_']],
+            'domain': [['name', 'in', ['delivery_bpost', 'delivery_dhl', 'delivery_easypost', 'delivery_fedex', 'delivery_ups', 'delivery_usps']]],
```

Comment:
i won't  do that
adding a carrier in enterprise would require a commit in community
use =like delivery_
!= 'delivery_barcoee'